src/components/Map/__tests__/MapWithGoogleLoader.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { MapWithGoogleLoader } from '..';
+import '@testing-library/jest-dom';
+
+describe('<MapWithGoogleLoader/> that loads google api and renders a Map', () => {
+    it('should render with default props', () => {
+        // @ts-ignore
+        const view = render(<MapWithGoogleLoader apiKey="someKey" />);
```

Comment:
After migrate Map to TS, it asks to provide all props 

![image](https://github.com/textkernel/oneui/assets/104063210/018f335a-5be0-45c4-956c-468668eeeb26)

Reply:
These are not props, but react component attributes. Looks to me, as the import was somehow wrong, and this file thinks that `MapWihtGoogleLoader` is not a component

Reply:
Could you please switch to my branch and open MapWithGoogleLoader.spec.tsx just delete // @ts-ignore

Reply:
I've added for map component
![image](https://github.com/textkernel/oneui/assets/104063210/abacd4ca-1041-4ef9-8bd8-ec4c7230171c)

Reply:
I see. I think there is a problem with how you defined the props in `src/components/Map/MapWithGoogleLoader/MapWithGoogleLoader.tsx`. You extended `GoogleMaps` here as well, but you rendering `Map`, so this is not consistent. I added some comments in the code to fix it.